services/src/main/java/io/druid/cli/CliPeon.java

```diff
@@ -190,7 +190,7 @@ public void configure(Binder binder)
             binder.bind(JettyServerInitializer.class).to(QueryJettyServerInitializer.class);
             Jerseys.addResource(binder, QueryResource.class);
             LifecycleModule.register(binder, QueryResource.class);
-            LifecycleModule.register(binder, LookupReferencesManager.class);
+            binder.install(new LookupModule());
```

Comment:
Do these bindings need to be *HERE* or can they be next in the list of modules. See how `
        new IndexingServiceFirehoseModule(),
        new ChatHandlerServerModule()` are used in CLIPeon